api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -198,7 +231,10 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	 */
 	public ConceptDatatype getConceptDatatype(Integer i) throws DAOException;
 	
+<<<<<<< HEAD
 	/**
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
These lines look like left-overs from Git conflicts. The should be removed.

Reply:
Ha ha  , true  , run into some conflicts which am resolving , i wanted to remove the commit history from the previous Ticket , that's why i run into these residues. Cleaning this out .Otherwise did you find the remaining enhancements fine.